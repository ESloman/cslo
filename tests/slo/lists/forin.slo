var a = ["a", "b", "c", "d", "e"];

for (var letter in a) {
    println(letter);
}

println(a);
