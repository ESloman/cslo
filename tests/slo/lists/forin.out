a
b
c
d
e
list[5]: [a, b, c, d, e]
